app/services/email_manager/mega_gam_receipt_sender.rb

```diff
@@ -0,0 +1,39 @@
+# frozen_string_literal: true
+
+module EmailManager
+  class MegaGamReceiptSender < BaseService
+    attr_reader :amount, :campaign_name, :payment_intent
+
+    def initialize(params)
+      @amount = params[:amount]
+      @campaign_name = params[:campaign_name]
+      @payment_intent = params[:payment_intent]
+    end
+
+    # rubocop:disable Layout/LineLength
+    def call
+      amount_string = EmailManager::Sender.format_amount(amount: amount)
+      html = '<!DOCTYPE html>' \
```

Comment:
tbd on the actual copy we want here. i'm punting this until we start testing / really need it

https://trello.com/c/8YWnoqHy/724-megagam-clarify-receipt-copy